helm/private/helm_package.bzl

```diff
@@ -272,3 +223,33 @@ helm_package = rule(
         str(Label("@rules_helm//helm:toolchain_type")),
     ],
 )
+
+def helm_package(name, chart = None, chart_json = None, values = None, values_json = None, **kwargs):
+    """Creates a helm package.
+
+    Args:
+        name: The name of the target.
+        chart: The `Chart.yaml` file of the helm chart.
+        chart_json: The `Chart.yaml` file of the helm chart as a json object.
+        values: The `values.yaml` file for the current package.
+        values_json: The `values.yaml` file for the current package as a json object.
+        **kwargs: Additional arguments to pass to the underlying `helm_package` rule.
+    """
+    if chart_json:
```

Comment:
This can all be handled within the rule. `json_to_yaml` could instead be a macro that spawns the conversion action. The helm_pkg rule has more attributes but really that's what users see as the interface anyway.

Reply:
Actually if you do this then https://github.com/abrisco/rules_helm/pull/47#discussion_r1466595908 is not an issue anymore

Reply:
Ok, I made that change.